include/z64camera.h

```diff
@@ -0,0 +1,407 @@
+#ifndef Z64CAMERA_H
+#define Z64CAMERA_H
+
+#include "prevent_bss_reordering.h"
```

Comment:
This might be causing your bss issues. I think it's best if this only gets used in the `c` files that have reorderings.

Reply:
Right, I forgot about this. This line actually solved a really tough bss issue previously, where 20+ files were having bss issues and nothing seemed to help. Removing this does actually solve this case of bss plus it's as you said, better to have these in just the c-files. This was just a temp fix.

Of course I think the build still failed from an issue on the Jenkins side of things?